tests/utils.ts

```diff
@@ -172,7 +172,6 @@ async function setupGlobalAgent(
 }
 
 function generateRandomNodeId(): NodeId {
-  // Make it easy to read with an identifier
   const random = keysUtils.getRandomBytesSync(16).toString('hex');
   return IdInternal.fromString<NodeId>(random);
 }
```

Comment:
This is still generating only 16 bytes. Our NodeIds are 32 bytes. I believe this needs to be fixed.

I removed the comment as it is no longer relevant. @tegefaulkes please make the change and fix up any tests.

Reply:
This should be `IdInternal.fromBuffer<NodeId>(random);` where `random` should be `32` bytes.

Reply:
We're doing it this way so the raw string of the nodeId looks like a hex string.  So we can tell it apart when debugging. Just using the random bytes directly gives us a hard to read string.

Reply:
Ah right. Ok well just add a comment explaining that `hex` string is more readable, and that 16 bytes creates 32 hex digits.

Reply:
I'm leaving it as is for now.